sections/docker/docker-ignore.md

```diff
@@ -0,0 +1,48 @@
+# Use .dockerignore to prevent leaking secrets
+
+<br/><br/>
+
+### One Paragraph Explainer
+
+The Docker build command copies the local files into the build context environment over a virtual network. Cautious, development and CI folders contain secrets like .npmrc, .aws, .env files, and other sensitive files. Consequently, Docker images might hold secrets and expose them in unsafe territories (e.g. Docker repository, partners servers). In a better world, the Dockerfile should be explicit about what is being copied. On top of this, include a .dockerignore file that acts as the last safety net that filters out unnecessary folders and potential secrets. Doing so also boosts the build speed - By leaving out common development folders that have no use in production (e.g. .git, test results, IDE configuration), the build can better utilize the cache and achieve better performance
+
+<br/><br/>
+
+### Code Example – A good default .dockerignore for Node.js apps
+
+<details>
+<summary><strong>.dockerignore</strong></summary>
+
+```
+**/node_modules/
+**/.git
+**/README.md
+**/LICENSE
+**/.vscode
+**/npm-debug.log
+**/coverage
+**/.env
+**/.editorconfig
+**/.aws
+```
```

Comment:
what about including the `dist` or `build` folder? Nowadays it seems to me that most people build inside the docker image, so there is no need to copy the code you built previously on your local machine

Reply:
@kevynb Do you mean excluding the 'src'?

Reply:
No, I mean that we might want to add `dist` and `build` folders in the `docker-ignore` file. I guess it's better to have the `src` folder is usually needed to build the code 😄  Sorry for not being clear enough 😉 

P.S. On a totally unrelated matter, I found that there's an emoji for the wind goddess or something similar while trying to add the wink: 🌬️

Reply:
@kevynb Yes, makes sense + comment to prevent confusions

🌬️